stdlib/nativeint.ml

```diff
@@ -49,10 +49,10 @@ let min_int = shift_left 1n (size - 1)
 let max_int = sub min_int 1n
 let lognot n = logxor n (-1n)
 
-let unsigned_to_int =
+let unsigned_to_int n =
```

Comment:
This looks like a typo that would make the code ill-typed due to the function now having two parameters. (Maybe you meant to remove `fun n ->` below, and this is okay because the `max_int` computation magically gets shared as a constant, or is cheap to recompute?)

Reply:
Right, I committed the buggy version of this file :-(  Will fix shortly.

Reply:
So, yes, I hesitated between precomputing the "max_int" value or recomputing it at every call.  It turns out that ocamlopt generates the same (optimal) code in both cases, something I did not expect. (Yay! for constant propagation during closure conversion.)  For bytecode, the version that precomputes is better.  So, that's the one I kept.